### PR TITLE
Address Manager

### PR DESCRIPTION
**Pull request by Hirobreak**

### CODE REVIEW

**Comment by Hirobreak** [comment]
Screen Shot 2020-07-03 at 5 03 14 PM

File: iOS-Email-Client/Libs/EventHandler.swift [hunk 1]
```diff
@@ -440,6 +444,37 @@ extension EventHandler {
         finishCallback(true, .Empty)
     }
     
+    func handleDefaultUpdate(params: [String: Any], finishCallback: @escaping (_ successfulEvent: Bool, _ item: Event.EventResult) -> Void){
+        UIUtils.deleteSDWebImageCache()
```
Comment (jorgeblacio) [inline iOS-Email-Client/Libs/EventHandler.swift:448]: ?

File: iOS-Email-Client/Libs/EventHandler.swift [hunk 2]
```diff
@@ -440,6 +444,37 @@ extension EventHandler {
...
+        guard let myAccount = DBManager.getAccountById(self.accountId) else {
+            finishCallback(true, .Empty)
+            return
+        }
+        DBManager.update(account: myAccount, defaultAddressId: event.aliasId)
+        finishCallback(true, .Empty)
+    }
+    
+    func handleAddressNameUpdate(params: [String: Any], finishCallback: @escaping (_ successfulEvent: Bool, _ item: Event.EventResult) -> Void){
+        UIUtils.deleteSDWebImageCache()
```
Comment (jorgeblacio) [inline iOS-Email-Client/Libs/EventHandler.swift:459]: ?

**Comment by Hirobreak** [comment]
Screen Shot 2020-07-06 at 10 28 51 AM

**Pull request merged**